src/biokbase/narrative/staging/helper.py

```diff
@@ -0,0 +1,121 @@
+
+import os
+import urllib
+import urllib2
+import json
+
+"""
+KBase staging.helper class
+"""
+__author__ = "Tianhao Gu <tgu@anl.gov>"
+
+
+class Helper(object):
+
+    def __token(self):
+        """
+        Authorization token passed in from front-end.
+        """
+
+        token = os.environ.get('KB_AUTH_TOKEN')
+
+        if not token:
+            raise ValueError('Cannot retrieve auth token')
+
+        return token
+
+    def __staging_url(self):
```

Comment:
URLs should be added to `src/config.json`

They can be fetched with the `biokbase.common.url_config` class as:
```from biokbase.narrative.common.url_config import URLS
my_url = URLS.key_from_config_file
```

Actually, I just checked and the staging service is already there. You can fetch that url with:
```
staging_url = URLS.staging_api_url
```

It will automatically know what environment you're in, too.

Reply:
👍